src/components/controls/TextArea/TextArea.tsx

```diff
@@ -47,7 +49,8 @@ export const TextArea = React.forwardRef<HTMLSpanElement, TextAreaProps>(
             name,
             value,
             defaultValue,
-            disabled = false,
+            disabled: originalDisabled,
+            readOnly: originalReadOnly,
```

Comment:
I suggest to change ~~`originalDisabled`~~ -> `disabledProps` and ~~`originalReadOnly`~~ -> `readOnlyProp` like it did for `errorMessage` property. Just for code consistency

Reply:
[done](https://github.com/gravity-ui/uikit/pull/1747/commits/35f8f6bf6476e7102b1d139c0e7db75a9796a325)